### ISSUE TITLE
Panic on wasm

### ISSUE THREAD

**Issue by dalance** [jsinger67/parol#357]
After updating to parol_runtime 0.24.0, the following panic occurs on wasm build.

```
veryl_wasm.js:241 panicked at library/std/src/sys/pal/wasm/../unsupported/time.rs:13:9:
time not implemented on this platform
```

It seems to be caused by the following dependency.
https://github.com/jsinger67/parol/blob/fa7b107f8d5261178953b77edc41fcef84450a6c/crates/parol_runtime/src/lexer/tokenizer.rs#L1

Could you remove this or add `#[cfg(not(target_family = "wasm"))]`?

**Comment by jsinger67** [comment]
Fixed with 63105f56
I released parol_runtime 0.24.1 and hope this remedies the problems with wasm.
@dalance, please reopen the issue in case of other problems.

**Comment by dalance** [comment]
Thanks!
It works fine.
